build.gradle

```diff
@@ -31,9 +31,9 @@ dependencies {
     compile("org.springframework.boot:spring-boot-starter-web:${springBootVersion}")
     compile("org.springframework.boot:spring-boot-starter-thymeleaf:${springBootVersion}")
     compile('org.springframework.boot:spring-boot-starter-security')
-    compile('org.springframework.security.oauth.boot:spring-security-oauth2-autoconfigure:2.0.1.RELEASE')
-    compile('org.springframework.security.oauth:spring-security-oauth2:2.2.0.RELEASE')
-    compile('com.google.apis:google-api-services-sheets:v4-rev516-1.23.0')
+    compile("org.springframework.security.oauth.boot:spring-security-oauth2-autoconfigure:${springBootVersion}")
```

Comment:
not sure if this always the same version like spring boot version

but as far as I understand this dependency will be removed in the next step completely, so does not matter?

Reply:
I upgraded in several steps (by gradually updating spring boot version) and checked how version of this lib correlates with used spring boot version, it was always 1-to-1.